stc_game/Assets/Scripts/SceneControl/PersistentScene/PersistentScene.cs

```diff
@@ -10,31 +10,83 @@
 
 public class PersistentScene : MonoBehaviour
 {
-    public static PersistentScene Instance;
+    public static PersistentScene Instance = null;
     public User user;
-    public UserService userService = UserService.Instance;
     public GameCharacter GameCharacter;
     public ReviveController reviveController;
     public ActionBarController actionBar;
     public QuestWindowUI questWindowUI;
     public DialogueUI dialogueUI;
     public HUDController hud;
+    public Button logoutButton;
+    public Button exitButton;
+    public LogoutCanvas logoutCanvas;
+
     private void Awake()
     {
         if (Instance == null)
         {
-            DontDestroyOnLoad(gameObject);
             Instance = this;
+            DontDestroyOnLoad(gameObject);

```

Comment:
updated the format of a dontdestroyonload according to a newer template recommendation